engine/admin/systemd.md

```diff
@@ -182,16 +182,17 @@ you will need to add this configuration in the Docker systemd service file.
     $ sudo systemctl daemon-reload
     ```
 
-5.  Verify that the configuration has been loaded:
+5.  Restart Docker:
```

Comment:
Are you sure this is needed? I don't think so, because the `daemon-reload` above will send a HUP to the daemon, which will make it re-read its configuration. Did you test to make sure it's needed?